.gitignore

```diff
@@ -27,3 +27,5 @@ coverage
 *.sw?
 
 *.tsbuildinfo
+
+dist/
```

Comment:
Why ignore the dist? I wonder if it might break the pages build

Reply:
Oops, I didn't realize this was for Pages. Re-added.